Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/Items/AddModuleNowMenu.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace Dnn.EditBar.UI.Items
+{
+    using System;
+    using System.Linq;
+
+    using Dnn.EditBar.Library;
+    using Dnn.EditBar.Library.Items;
+    using DotNetNuke.Entities.Portals;
+    using DotNetNuke.Services.Personalization;
+    using DotNetNuke.Web.Components.Controllers;
+
+    [Serializable]
+    public class AddModuleNowMenu : BaseMenuItem
+    {
+        private const string LocalResourceFile = "~/DesktopModules/Admin/Dnn.EditBar/App_LocalResources/AddModuleNow.resx";
+
+        public override string Name { get; } = "AddModuleNow";
+
+        public override string Text
+        {
+            get
+            {
+                return DotNetNuke.Services.Localization.Localization.GetString("AddModuleNow", LocalResourceFile, System.Threading.Thread.CurrentThread.CurrentCulture.Name);
+            }
+        }
+
+        public override string CssClass
+        {
+            get
+            {
+                return string.Empty;
+            }
+        }
+
+        public override bool CustomLayout => true;
+
+        public override string Template
+        {
+            get
+            {
+                var portalSettings = PortalSettings.Current;
+                if (portalSettings == null)
+                {
+                    return string.Empty;
+                }
+
+                var moduleList = ControlBarController.Instance.GetCategoryDesktopModules(portalSettings.PortalId, "All", string.Empty).Select(m =>
+                {
+                    var selected = m.Value.FriendlyName == "HTML" ? " selected" : string.Empty;
+                    return $"<option value=\"{m.Value.DesktopModuleID}\"{selected}>{m.Value.FriendlyName}</option>";
+                });
+                var panes = string.Empty;
+                foreach (string paneName in portalSettings.ActiveTab.Panes)
+                {
+                    var selected = paneName == "ContentPane" ? " selected" : string.Empty;
+                    panes += $"<option value=\"{paneName}\"{selected}>{paneName}</options>";
+                }
+
+                var toolTip = DotNetNuke.Services.Localization.Localization.GetString("AddModuleNow.Tooltip", LocalResourceFile, System.Threading.Thread.CurrentThread.CurrentCulture.Name);
```

Comment:
Is there a need to explicitly pass the current culture?  Shouldn't that happen by default?